src/form/src/components/styled/StyledForm.js

```diff
@@ -93,6 +93,7 @@ const StyledForm = styled(Form)`
   .grid-container {
     display: flex;
     flex: 1;
+    flex-direction: column;
```

Comment:
a grid-container up until this point has only ever had one child, which is the grid. this column lets the attachments section sit below instead of to the side. All of the schemas in the stories appear unaffected by this change.